File: feature/src/main/java/com/ortin/gesturetranslator/feature/managers/camera/CameraManager.java
package com.ortin.gesturetranslator.feature.managers.camera;

import androidx.lifecycle.LifecycleOwner;

import com.ortin.gesturetranslator.feature.managers.camera.listeners.CameraListener;

public interface CameraManager {
    void loadImage(CameraListener cameraListener, LifecycleOwner lifecycleOwner);
    void setStatusFlashlight(boolean mode);
}

File: feature/src/main/kotlin/com/ortin/gesturetranslator/feature/managers/camera/CameraManager.kt
package com.ortin.gesturetranslator.feature.managers.camera

import androidx.lifecycle.LifecycleOwner
import com.ortin.gesturetranslator.feature.managers.camera.listeners.CameraListener

interface CameraManager {
    fun loadImage(cameraListener: CameraListener, lifecycleOwner: LifecycleOwner, cameraFacing: Int)
    fun setStatusFlashlight(mode: Boolean)
}
